### ISSUE TITLE
Discrepancy code/paper about the L_simple loss

### ISSUE THREAD

**Issue by Mathux** [Stanford-TML/EDGE#20]
Hello,

First of all, thanks for your nice work. 

I am trying to understand the training losses, and there is something I don't understand.
In the paper (equation 2, page 3), it is written that you are using the loss L_simple (L2 between x_gt and x_generated).

But when I looked at the code, it seems that this loss is multiplied by the "p2_loss_weight" term:
https://github.com/Stanford-TML/EDGE/blob/17c3428669ed6733edd9d8c66f7dc62060b8e46d/model/diffusion.py#L464

which is defined here:
https://github.com/Stanford-TML/EDGE/blob/main/model/diffusion.py#L127-L131

Can you explain why and how the loss is scaled depending on the time step? I did not find this information in the paper.

Thanks for your help!

-- 
Mathis

**Comment by Mathux** [comment]
Hello,

I just found that the parameter "use_p2" is set to False, which means that the weights are all 1s. (So the multiplication does not do anything).

So all good, I am closing the issue :)
